### DISCUSSION TITLE
Create a template for simple apps to deploy on Kubernetes t21-hack-kube-apps

### DISCUSSION THREAD

**Discussion by filippo82** [softwareunderground/transform-2021-hackathon#3] [Projects]
## Slack channel
[t21-hack-kube-apps](https://swung.slack.com/archives/C01RY1XLHAQ)

## People
* Filippo [Squirro]

## Description
I will set up a Kubernetes cluster on AWS and I would like to work towards building a *template* for an app than can be easily deployed on this cluster using GitOps best practices. Ideally, this template could be a `yaml` file which needs to validate against a schema.

## Goals
* Deploy an app by the end of Transform 21 using pull requests on GitHub

## Resources
* TBD

**Comment by RichardScottOZ** [comment]
https://stackstac.readthedocs.io/en/latest/
